st2api/tests/unit/controllers/v1/test_executions_rbac.py

```diff
@@ -135,3 +138,74 @@ def test_get_all_limit_minus_one(self):
 
         resp = self.app.get('/v1/actionexecutions?limit=-1')
         self.assertEqual(resp.status_code, http_client.OK)
+
+    def test_get_respective_actions(self):
```

Comment:
Let's please also add test cases for the following scenarios:

* system user can view all
* admin user can view all
* any user can view all if rbac is disabled
* any user can view all if rbac is enabled and permission isolation is disabled

Reply:
will update in next commit.